kinto/config/__init__.py

```diff
@@ -1,46 +1,46 @@
 import os
 import binascii
 import codecs
+from six.moves import input
 
 HERE = os.path.abspath(os.path.dirname(__file__))
 
+def render_template(template, destination, values):
+   
+   here = os.path.abspath(os.path.dirname(__file__))
+   template = os.path.join(here, 'kinto.tpl') 
+   destination = os.path.join(here, 'kinto.ini')
 
-def render_template(template, destination, **kwargs):
-    template = os.path.join(HERE, template)
```

Comment:
yes, the only think you might want to change here is the mode which should be "w" instead of "w+".